CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- Fix slow and frozen frames tracking ([#2271](https://github.com/getsentry/sentry-java/pull/2271))
```

Comment:
@marandaneto since no public API changes were necessary I assume this fix doesn't require any further action on hybrid SDKs.